.github/workflows/python.yaml

```diff
@@ -2,9 +2,15 @@ name: Python
 
 on:
   push:
-    branches: [main, autoupdate/strict]
+    branches:
+      - main
+      - autoupdate/strict
+      - 'release-[0-9]+.[0-9]+'
+      - 'release-[0-9]+.[0-9]+-strict'
   pull_request:
-    branches: [main]
+    branches:
+      - main
+      - 'release-[0-9]+.[0-9]+'
```

Comment:
with this, we won't have to change the github actions across branches, right? nice!

Reply:
righto.  the actions never have to change